test/performance/tools/update-serving/update-test.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
```

Comment:
this uses a different image than the one in the cron-job. I want a way to test the changes to these scripts before applying it to cron

Reply:
I'd like to see a better test story for testing these in general because bad runs will really pollute our trend lines.  I'm fine with leaving this for now, but all of this should change to `ko`-based, and `KO_DOCKER_REPO` can address your namespacing concerns.  Eliminating this can wait until that switch, but we need to make that switch and have a way to canary things.

Reply:
Agreed. We need to have a better story for locally testing these. Getting this in for now.

Reply:
I'm not even just talking about local testing, but we can quibble later :)